ciao-deploy/deploy/setup.go

```diff
@@ -200,6 +200,7 @@ Type=simple
 ExecStart=/usr/local/bin/%s --cacert=%s --cert=%s --v 3
 Restart=no
 KillMode=process
+TasksMax=infinity
```

Comment:
I guess we only need this for launcher.   I have a patch that I'm developing for the running as non-root issue that allows us to customize these unit files.  We could merge this for now and I can make this launcher only in my patch.

Reply:
@markdryan I wonder if this could also happen in ciao-controller. It starts a new goroutine for each instance and although they are not long lived (for sending messages by ssntp) there is a good chance the number of alive and blocked ones will be quite high if a large number of instances are created simultaneously

Reply:
I'd thought TaskMax was related to creation of new processes via forks and not the creation of new threads.  However, thinking about it, controller does launch processes when creating images and snapshots, so it needs this too particularly if it does this in parallels.  Let's merge it.

Reply:
FYI - this does happen on controller due to forks for snapshot/volume deletion (try launching 2000 instances, then typing ciao-cli instance delete -all)